src/ipc/handler.c

```diff
@@ -1245,8 +1245,9 @@ int ipc_queue_host_message(struct ipc *ipc, uint32_t header, void *tx_data,
 			   size_t tx_bytes, bool replace)
 {
 	struct ipc_msg *msg = NULL;
-	uint32_t flags, found = 0;
+	bool found = false;
```

Comment:
"bool replaced = false" perhaps? Ok with found as well, just a bit confusing to read the function body.